dse-db/test/unit/org/apache/cassandra/index/sai/disk/KDTreeIndexSearcherTest.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+
+package org.apache.cassandra.index.sai.disk;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Iterators;
+import com.google.common.collect.Lists;
+import org.junit.Test;
+
+import org.apache.cassandra.db.marshal.AbstractType;
+import org.apache.cassandra.db.marshal.DecimalType;
+import org.apache.cassandra.db.marshal.Int32Type;
+import org.apache.cassandra.db.marshal.LongType;
+import org.apache.cassandra.db.marshal.NumberType;
+import org.apache.cassandra.db.marshal.ShortType;
+import org.apache.cassandra.index.sai.SAITester;
+import org.apache.cassandra.index.sai.SSTableQueryContext;
+import org.apache.cassandra.index.sai.Token;
+import org.apache.cassandra.index.sai.plan.Expression;
+import org.apache.cassandra.index.sai.utils.NdiRandomizedTest;
+import org.apache.cassandra.index.sai.utils.TokenFlow;
+import org.apache.cassandra.utils.CloseableIterator;
+import org.apache.cassandra.utils.flow.Flow;
+
+public class KDTreeIndexSearcherTest extends NdiRandomizedTest
```

Comment:
Did this test get duplicated by accident? Should we have only one KDTreeIndexSearcherTest located under `datastax-cassandra/test/unit/org/apache/cassandra/index/sai/disk/`?

If so, I can clean this up as part of my effort in updating the datastax:storage_attached_index